Cargo.toml

```diff
@@ -29,8 +29,8 @@ pretty_env_logger = { version = "0.4", optional = true }
 anyhow = "1.0"
 filetime = "0.2.9"
 flate2 = { version = "1.0.3", default-features = false, features = ["zlib"] }
-git2 = "0.13.23"
-git2-curl = "0.14.1"
+git2 = "0.14.1"
```

Comment:
Is there any particular reason this and libgit2-sys reference an older version?  I seem to recall that `.1` was broken.